### PR TITLE
HPCC-25720 - First Portuguese version of 8.0 xml files

### PR DESCRIPTION
**Pull request by vfumene**
…eview

Signed-off-by: RISK\fumeva01 <valdir.fumene@lexisnexisrisk.com>

## Type of change:
- [x ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [ x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: docs/PT_BR/CMakeLists.txt [hunk 1]
```diff
@@ -15,10 +15,10 @@
 ################################################################################
 
 ###
-#  This CMakeLists intended to be used in docs/PT_BR
+#  This CMakeLists intended to be used in docs/EN
 #    Refactored for internationalization
 ###
-#    Last Mod GP 5/2019
+# Last Mod GP 7/2018
```
Comment (g-pan) [inline docs/PT_BR/CMakeLists.txt:21]: No reason to change this - but can be changed to 2021 if necessary

File: docs/PT_BR/CMakeLists.txt [hunk 2]
```diff
@@ -27,14 +27,14 @@
 
 
 cmake_minimum_required(VERSION 2.8)
-PROJECT(docs_pt_br)
+PROJECT(docs_en_us)
```
Comment (g-pan) [inline docs/PT_BR/CMakeLists.txt:30]: please revert value back to (docs_pt_br)

File: docs/PT_BR/CMakeLists.txt [hunk 3]
```diff
@@ -27,14 +27,14 @@
...
 
-#define_property(GLOBAL PROPERTY DOC_TARGETS BRIEF_DOCS "docs_pt_br" FULL_DOCS "docs_pt_br")
+#define_property(GLOBAL PROPERTY DOC_TARGETS BRIEF_DOCS "docs_en_us" FULL_DOCS "docs_en_us")
```
Comment (g-pan) [inline docs/PT_BR/CMakeLists.txt:32]: please revert value back to (docs_pt_br)

File: docs/PT_BR/CMakeLists.txt [hunk 4]
```diff
@@ -27,14 +27,14 @@
...
 #mark_as_advanced(DOC_TARGETS)
 
 #include(${CMAKE_MODULE_PATH}/docMacros.cmake)
 
-set (DOC_LANG PT_BR)
+set (DOC_LANG EN_US)
```
Comment (g-pan) [inline docs/PT_BR/CMakeLists.txt:37]: please set value back to (PT_BR)

File: docs/PT_BR/CMakeLists.txt [hunk 5]
```diff
@@ -63,7 +63,7 @@ add_subdirectory(ECLProgrammersGuide)
 add_subdirectory(ECLStandardLibraryReference)
 add_subdirectory(ECLReference)
 
-##add_subdirectory(EclipseHelp)
+add_subdirectory(EclipseHelp)
```
Comment (g-pan) [inline docs/PT_BR/CMakeLists.txt:66]: leave this commented out

File: docs/PT_BR/CMakeLists.txt [hunk 6]
```diff
@@ -83,19 +83,17 @@ add_subdirectory(RunningHPCCinaVirtualMachine)
 add_subdirectory(ECLScheduler)
 add_subdirectory(ECLWatch)
 add_subdirectory(VisualizingECL)
-#9/16
 add_subdirectory(WsSQLUG)
-#add_subdirectory(PortalHTML)
-
+add_subdirectory(PortalHTML)
```
Comment (g-pan) [inline docs/PT_BR/CMakeLists.txt:87]: leave this commented out as before.

File: docs/PT_BR/CMakeLists.txt [hunk 7]
```diff
@@ -83,19 +83,17 @@ add_subdirectory(RunningHPCCinaVirtualMachine)
...
 
 # Docs automation
-##CommentedOut4PT_BR_TESTING
-##if (DOCS_AUTO)
+if (DOCS_AUTO)
 #   add_subdirectory(XMLGeneration)
-##   add_subdirectory(ConfiguringHPCC)
-##endif()
+   add_subdirectory(ConfiguringHPCC)
+endif()
```
Comment (g-pan) [inline docs/PT_BR/CMakeLists.txt:96]: please comment the above two lines as before.

**Review by g-pan — COMMENTED** [review]
Please see comments in line

**Comment by vfumene** [comment]
@g-pan please review

**Comment by richardkchapman** [comment]
Please fix the commit title so that it looks better in the changelog (the @gpan please review bit should not be there, and nor should the - after the Jira number)

**Comment by vfumene** [comment]
@g-pan could please review one more time?

**Review by g-pan — APPROVED** [review]
I can approve now that my unit testing has succeeded.

**Comment by richardkchapman** [comment]
Merged manually to correct commit message

**Pull request closed without merging**